packages/ng/libraries/core/src/style/components/_input.scss

```diff
@@ -1,4 +1,6 @@
 @import "~@lucca-front/scss/src/mixins/forms";
+@import "@angular/cdk/text-field-prebuilt.css";
```

Comment:
we dont want to import angular/cdk css

if you look at the style for the overlay [here](https://github.com/LuccaSA/lucca-front/blob/master/packages/ng/libraries/core/src/style/components/_overlay.scss) you'll see that we dont import their css but we use their selectors

Reply:
du coup, il faut intégrer le contenu de `@angular/cdk/text-field-prebuilt.css` dans LF ou charge aux BUs d'importer ce fichier ?

Reply:
english

the target is that all will be handled by LF, no need to import another third party stylesheet